packages/ui5-middleware-simpleproxy/lib/proxy.js

```diff
@@ -17,17 +25,20 @@ const log = require("@ui5/logger").getLogger("server:custommiddleware:proxy");
  * @returns {function} Middleware function to use
  */
 module.exports = function({resources, options}) {
+    const providedBaseUri = (options.configuration && options.configuration.baseUri) || env.baseUri;
+    const providedStrictSSL = (options.configuration && options.configuration.strictSSL) || env.strictSSL;
```

Comment:
Hmm, this looks a bit tricky here - it works fine if only one of those values (either config or env) is defined - but in case of both values are defined, it behaves odd, assume:

`config.SSL = false || env.SSL = true` ==> env wins
`config.SSL = true || env.SSL = false` ==> config wins

What is the expected precedence behavior here? Config before env or env before config. I assume the last one should be the right one as the env vars should overrule the configs, right? 

By using the `${env.XXX}` syntax there would be a clear distinction on what is used. In this case here you need to do `undefined checks` and then fallback to the other value, e.g.:

Quick and dirty: 😄 
```js
// only consider strictSSL from env when a real value is set otherwise the value is undefined
const env = {
  baseUri: process.env.UI5_MIDDLEWARE_SIMPLE_PROXY_BASEURI,
  strictSSL: (function(value) { return value !== undefined && !(value === "false")  })(process.env.UI5_MIDDLEWARE_SIMPLE_PROXY_STRICT_SSL)
}
```
 
For the provided checks I would suggest to prefer the `env` variable and only of the env variable is undefined we fallback to the config, e.g.:

```js
const providedStrictSSL = env.strictSSL !== undefined ? env.strictSSL : (options.configuration && options.configuration.strictSSL);
```

What do you think?